apps/sharedtest/test/unit/bluetooth_helper_v2_test.js

```diff
@@ -33,49 +33,52 @@ suite('bluetooth helper', function() {
     sandbox.spy(MockBTAdapter, 'toggleCalls');
     subject.toggleCalls();
     subject.toggleCalls();
+
```

Comment:
Sometimes, one line space for segment one assertion is a little bit strange for me. It's an optional suggestion to use `// assertion` instead of one line gap. Optional! XD